lib/components/form/call-taker/date-time-options.js

```diff
@@ -0,0 +1,183 @@
+import {
+  OTP_API_DATE_FORMAT,
+  OTP_API_TIME_FORMAT
+} from '@opentripplanner/core-utils/lib/time'
+import moment from 'moment'
+import React, { Component } from 'react'
+import { OverlayTrigger, Tooltip } from 'react-bootstrap'
+
+const departureOptions = [
+  {
+    // Default option.
+    value: 'NOW',
+    children: 'Now'
+  },
+  {
+    value: 'DEPART',
+    children: 'Depart at'
+  },
+  {
+    value: 'ARRIVE',
+    children: 'Arrive by'
+  }
+]
+
+const TIME_FORMATS = [
```

Comment:
Either add a comment for this constant, or rename to something more descrpitive.